f2/src/locales/en.json

```diff
@@ -86,6 +86,8 @@
   "contactinfoPage.skipButton": "Skip",
   "contactinfoPage.skipInfo": "Skip this step to report anonymously.",
   "contactinfoPage.title": "Your contact details",
+  "default.conjunction": ", and ",
```

Comment:
as discussed IRL we need another conjunction for the case where there are two items.